src/chains/ethereum/options/package-lock.json

```diff
@@ -1,326 +1,402 @@
 {
 	"name": "@ganache/ethereum-options",
 	"version": "0.8.0",
-	"lockfileVersion": 1,
+	"lockfileVersion": 3,
```

Comment:
I'd love to update to v3, but it should be all at once and in a different PR (the one that adds support for node 20, probably?)